src/zmq/zmqpublishnotifier.cpp

```diff
@@ -2,14 +2,25 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#if defined(HAVE_CONFIG_H)
+#include <config/bitcoin-config.h>
+#endif
```

Comment:
Is this required here as no defines from `bitcoin-config.h` is used here?

Reply:
I guess not, removed.